frontend/packages/component-library/src/formFieldWrapper/FormFieldWrapper.tsx

```diff
@@ -0,0 +1,102 @@
+import classNames from 'classnames';
+import {ReactNode, useMemo, HTMLAttributes} from 'react';
+
+import {ComponentSizeXSToL} from '@/common/types';
+import FontAwesomeV6Icon, {FontAwesomeV6IconProps} from '@/fontAwesomeV6Icon';
+
+import moduleStyles from './formFieldWrapper.module.scss';
+
+export interface FormFieldWrapperProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
I wonder if something like a hook/HoC that enhances a specific field would make sense in this situation. I'll defer to @levadadenys or @kelbyhawn.

I'm imagining that one can use `withFormField(TextField)` which applies this wrapper. I'd also be interested in how other component libraries handle this scenario.

Reply:
(What is a HoC in this case? I know it's not Hour of Code 😜)

Reply:
The reason why I bring it up is that it feels strange to have a 1st class component whose sole purpose is to wrap the a form field. HoC is [Higher order component ](https://legacy.reactjs.org/docs/higher-order-components.html)

Reply:
It's called Wrapper Component and it's just another way of doing practically the same task.
It follows existing patterns we use across dsco (see withTooltip). Moreover, it follows the patterns how people are using this functionality right now.

If needed, hoc can be added later as a follow up, but right now we have exactly what we need.